docs/docs/using-graphql-playground.md

```diff
@@ -4,7 +4,7 @@ title: Using the GraphQL Playground
 
 ## Intro
 
-In this page we want to introduce you to an alternative to the current IDE for your GraphQL queries: [GraphQL Playground](https://github.com/prisma/graphql-playground).
+On this page, we want to introduce you to an alternative to the current IDE for your GraphQL queries: [GraphQL Playground](https://github.com/prisma/graphql-playground).
```

Comment:
```suggestion
This page will introduce you to an alternative to the current IDE for your GraphQL queries: [GraphQL Playground](https://github.com/prisma/graphql-playground).
```
Might as well update this for the Style Guide.

Reply:
Thank you!